app/controllers/v1/tiles_controller.rb

```diff
@@ -0,0 +1,40 @@
+class V1::TilesController < ApplicationController
+  def create
+    tile = Tile.new(create_params)
```

Comment:
Okay, I think it is okay to wait, maybe add a TODO: comment here and a ticket so we don't miss it once those are in place.